Sources/PrimerSDK/Classes/PCI/Checkout Components/PrimerCardData.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  PrimerCardData.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 27/09/22.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+
+public class PrimerCardData: PrimerRawData {
```

Comment:
I don't think that has been modified at all, just moved, right?

Reply:
Correct